types/src/index.d.ts

```diff
@@ -182,8 +184,15 @@ export type EasPassport = {
   fee: any;
 };
 
+export type PassportAttestation = {
+  multiAttestationRequest: MultiAttestationRequest[];
+  recipient: string;
```

Comment:
I think the recipient does not need to be here, since it will always be included in the `MultiAttestationRequest.data`